source/extensions/omni.isaac.lab/omni/isaac/lab/sim/simulation_context.py

```diff
@@ -467,6 +474,9 @@ async def reset_async(self, soft: bool = False):
 
     def _init_stage(self, *args, **kwargs) -> Usd.Stage:
         _ = super()._init_stage(*args, **kwargs)
+        self.set_setting("/app/player/playSimulations", False)
```

Comment:
Can you add a comment above saying why this is now done or needed? Isn't `sim.reset()` also calling something similar?